### ISSUE TITLE
SVG/PNG imported in React component listed as Unlisted dependencies

### ISSUE THREAD

**Issue by KirianCaumes** [webpro/knip#376]
Hi,

I have a React component (with Vite) that imports an image like this:
```tsx
import MySvg from 'assets/svg/my.svg'

export default function Example() {
    return (
        <>
            Some text
            <MySvg />
        </>
    )
}
```

When I run knip, this import is found in `Unlisted dependencies`
```sh
Unlisted dependencies (1)
assets/svg/my.svg                      src/components/elements/example.component.tsx  
```

I tried to hide it by adding the image path to `ignoreDependencies`, but that does not work.

```sh
Configuration issues (1)
Unused item in ignoreDependencies: assets/svg/my.svg
```

Is there a way to handle that this of element?

Thank you!

**Comment by EvgenyOrekhov** [comment]
This is a regression. I'm still using knip 2.41.3 and it works fine, but the latest version (3.3.5) has this issue.

**Comment by EvgenyOrekhov** [comment]
I was able to fix it in my project by declaring `@` as an alias for `src`

I added this to `tsconfig.json`

```jsonc
  "compilerOptions": {
    // ...
    "baseUrl": "src",
    "paths": {
      "@/*": ["*"]
    }
  }
```

and this to webpack config

```js
    alias: {
      '@': path.resolve(__dirname, 'src'),
    },
```

So my imports now look like `import MySvg from '@/assets/svg/my.svg'`
No more false warnings from knip.

**Comment by webpro** [comment]
Tbh I think `@` could be confusing too, since `@scoped/packages` exist (and also e.g. `@scoped/packages/file` is a valid specifier). But yeah, `@/` is different and works just fine.

I just realized one thing you could also do today that fixes the issue in the attached repository:

```ts
{
  compilers: {
    svg: () => '',
    png: () => ''
  }
}
```

Then we trick TS into thinking `'assets/svg/my.svg.ts'` exists. Then you don't need to change anything else in the codebase.

Perhaps I should try to build this into Knip (again)...

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.4.0. See [Release 3.4.0](https://github.com/webpro/knip/releases/tag/3.4.0) for release notes._

**Comment by webpro** [comment]
...so instead of changing anything, we can now also install v3.4.0 :)

Please let me know if there are any issues or regressions!
